product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml

```diff
@@ -0,0 +1,281 @@
+product: EDB Postgres Distributed
+version: 5.6.0
+date: 15 October 2024
+intro: |
+ EDB Postgres Distributed 5.6.0 includes a number of enhancements and bug fixes.
+ Please add more text here.
+highlights: |
+ - Streaming Transaction support with SDW.
```

Comment:
Not sure if this is highlight tbh, given we have one user of SDW, but we don't ave mch highlights, so dunno, but certainly not the top one